shipment_planning_maxoptra/models/res_partner.py

```diff
@@ -15,6 +15,16 @@ class ResPartner(models.Model):
         "partner to set on Batch pickings after import.",
     )
 
+    partner_key = fields.Text(string="Key")
```

Comment:
This field name and string must be more explicit
```suggestion
    maxoptra_partner_key = fields.Text(string="Maxoptra Key")
```